frontend/src/bundles/talent-onboarding/components/candidate-profile/candidate-parameter/candidate-parameter.tsx

```diff
@@ -0,0 +1,44 @@
+import { Typography } from '~/bundles/common/components/components.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+import { type CandidateIcons } from '~/bundles/talent-onboarding/enums/candidate-icons.enum.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    icon: ValueOf<typeof CandidateIcons>;
+    imgAlt: string;
+    text: string | number | string[];
+    className?: string;
+    typographyVariant?:
```

Comment:
why do we need all these options?

Reply:
@nikita-remeslov 
because inside this component we need to fill img alt = imgAlt prop, Typography variant = typographyVariant prop and text which will be displayed in Typography = text prop and className will change the style of Typography.

Reply:
I will remove imgAlt because I will replace current icons with material UI icons